MAUI/DataForm/DataForm-settings.md

```diff
@@ -0,0 +1,899 @@
+
+{% tabs %}
+{% highlight C# %}
+
+    
+    [DataFormDisplayOptions(ShowLabel = false)]
+    public string MiddleName { get; set; }
+    
+}
+{% endhighlight %}
+{% endtabs %}
+
+#### Using events
+
+You can hide the label text for editor by using the `ShowLabel` property in `DataFormItem`.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "FirstName")
+            {
+                e.DataFormItem.ShowLabel = false;
+            }
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+GenerateDataFormItem event when `AutoGenerate` is true.
+
+For `AutoGenerate` false you can use property directly by using the DataFormItem - add reference link for explicit creaate data editor
+
+## Change editor layout settings
+
+You can change the label width, label position and editor width. This is possible by using the `GenerateDataFormItem` event when `AutoGenerate` is true. In the event, change the layout by using the `DefaultLayoutSettings` property in the `DataFormItem`.
+
+For `AutoGenerate` false you can use property directly by using the DataFormItem - add refrence link for explict create data editor.
+
+By default label position is left.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "Description")
+            {
+                e.DataFormItem.DefaultLayoutSettings.LabelPosition = DataFormLabelPosition.Top;
+                e.DataFormItem.DefaultLayoutSettings.LabelWidth = 100;
+                e.DataFormItem.DefaultLayoutSettings.EditorWidth = 200;
+            }
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+GenerateDataFormItem event when `AutoGenerate` is true.
+
+For `AutoGenerate` false you can use property directly by using the DataFormItem - add reference link for explicit creaate data editor
+
+## Change the Background for editor layout
+
+You can change the background color for the label and editor by using `GenereateDataFormItem` event when `AutoGenerate` is true.
+
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "FirstName")
+            {
+                e.DataFormItem.Background = Brush.Azure; 
+            }
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+GenerateDataFormItem event when `AutoGenerate` is true.
```

Comment:
GenerateDataFormItem event when `AutoGenerate` is true.

For `AutoGenerate` false you can use property directly by using the DataFormItem - add reference link for explicit creaate data editor

remove

Reply:
removed